### ISSUE TITLE
Auth on website and game, kicking user off when on both

### ISSUE THREAD

**Issue by j3fffff** [StrangeLoopGames/EcoIssues#17904]
website and game share same auth and token system.
should be sperated, so user is not kicked form game, when logging into website

**Comment by denysaw** [comment]
Ok. I just won't delete existing accessToken of user. Should solve the problem.

**Comment by denysaw** [comment]
Hm. Though I see the issue is blocked. @redwyre Should I wait for your OAuth changes?

**Comment by j3fffff** [comment]
> Ok. I just won't replace existing accessToken of user. Should solve the problem.

wouldnt that break the function of auth to only allow one user logged in at a time?
might need seperate tokens for website and game?

**Comment by denysaw** [comment]
> 
> 
> > Ok. I just won't replace existing accessToken of user. Should solve the problem.
> 
> wouldnt that break the function of auth to only allow one user logged in at a time?
> might need seperate tokens for website and game?

Yeah, it would )) I didn't know had such limitation ) But is it problem, that user could log into the website in 2 browsers/devices?
He won't be able to login 2 times in-game as Auth will refresh token anyway. But yeah, separate tokens seem like ideal solution 👍🏻  Thanx for the idea ;)
